src/app/feed/feed-create-post/feed-create-post.component.ts

```diff
@@ -71,6 +72,25 @@ export class FeedCreatePostComponent implements OnInit {
     this._setRandomMovieQuote();
   }
 
+  emojiClick(event: any): void {
+    const trigger = event.currentTarget;
+    const picker = new EmojiButton({
+      theme: window.localStorage.getItem("theme") == "light" ? "light" : "dark",
+      position: "auto",
+      autoHide: false,
+      emojiSize: "1.3em",
+      emojisPerRow: 6,
+      rows: 3,
+      initialCategory: "smileys",
+      showPreview: false,
```

Comment:
let's set z-index here actually and just make it greater than the z-index of the global__top-bar